Source/ASCollectionView.mm

```diff
@@ -869,17 +846,29 @@ - (CGSize)_sizeForUIKitCellWithKind:(NSString *)kind atIndexPath:(NSIndexPath *)
   return size;
 }
 
+- (void)_superReloadData:(void(^)())updates completion:(void(^)(BOOL finished))completion
+{
+  if (updates) {
```

Comment:
Seems like the `updates` block should be non-nil. At the very least, we want to swap the maps. If it's non-nil, we can mark it that way and skip the nil check? The main point is I don't want people to think that it can ever be nil.

Reply:
It can be nil, if you just would like to call `[super reloadData]` with a completion block. I actually replaced all places where we call `[super reloadData]` with the call to `_superReloadData` so we have one central point to call in.

Reply:
Oh yeah, I overlooked those calls. 👍